src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -0,0 +1,165 @@
+from panoptes.pocs.focuser.serial import AbstractSerialFocuser
+
+
+class Focuser(AbstractSerialFocuser):
+    """
+    Focuser class for control of a Canon DSLR lens via an Astromechanics Engineering Canon EF/EF-S adapter.
+
+    Args:
+        name (str, optional): default 'Astromechanics Focuser'
+        model (str, optional): default 'Canon EF/EF-S'
+
+    Additional positonal and keyword arguments are passed to the base class, AbstractFocuser. See
+    that class' documentation for a complete list.
+
+    Min/max commands do not exist for the astromechanics controller, as well as
+    other commands to get serial numbers and library/hardware versions. However,
+    as they are marked with the decorator @abstractmethod, we have to override them.
+    """
+
+    def __init__(self,
+                 name='Astromechanics Focuser Controller',
+                 model='astromechanics',
+                 *args, **kwargs):
+        super().__init__(name=name, model=model, *args, **kwargs)
+        self.logger.debug('Initialising Astromechanics Lens Controller')
+
+        self._serial_port.baudrate = 38400
```

Comment:
Don't think this will work as `self._serial_port` is created only when `_connect` is run, which is hard-wired to `self._serial_port.baudrate = 115200`. I guess you haven't tested this code on the hardware?

My hope with adding this line was to move `def _connect(self, port):` into `serial.py` as it is mostly the same code.

You could move it there and add a required arg of `baudrate`, so: `def _connect(self, port, baudrate):`.

And then call it from birger or astromech like: `super().__init__(name=name, model=model, baudrate=38400, *args, **kwargs)` in `__init__` of `astromechanic.py`. 

Then modify `serial.py` to call `self.connect(self.port, baudrate)`

Be sure to delete `_connect` definition from `birger.py`

Reply:
It is working with the astro mechanics hardware. I was able to move a small lens and one of the Huntsman lenses.